### PR TITLE
upgrade sbt, scala and h2 dependency

### PR DESCRIPTION
**Pull request by kwark**
upgrade of:

- sbt to 1.0.4
- scala to 2.12.4
- h2 db driver to 1.4.196

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=1.0.2
+sbt.version=1.0.3
```
Comment (marcospereira) [inline project/build.properties]: sbt 1.0.4 is out. Do you mind updating to use it instead?

**Review by marcospereira — COMMENTED** [review]
Thank you, @kwark. See my comment about sbt 1.0.4.

Besides that, this LGTM.

**Comment by kwark** [comment]
@marcospereira Thanks for the review, requested upgrade to sbt 1.0.4 is included

**Review by marcospereira — APPROVED** [review]
LGTM.

Thank you, @kwark.

**Pull request merged**